src/menu.ts

```diff
@@ -137,6 +137,15 @@ const applicationMenu: MenuItemConstructorOptions[] = [
           config.set(ConfigKey.LaunchMinimized, checked)
         }
       },
+      {
+        label: 'Hide Menu bar',
```

Comment:
I think it would make sense to add this to the appearance menu.  @timche Thoughts?

Reply:
You're right, I'll move it there then.